src/backend/marsha/core/permissions.py

```diff
@@ -172,3 +172,62 @@ def has_permission(self, request, view):
             return True
 
         return super().has_permission(request, view)
+
+
+class StaffUserPermission(permissions.BasePermission):
+    """A custom permission class for API Tokens related to objects linked to a video.
```

Comment:
If it's linked to a video video, it should be also reflected in the class name or we will use it for other purpose otherwise.